README.md

```diff
@@ -230,6 +256,15 @@ Example:
 }
 ```
 
+## Building
```

Comment:
We try to keep commits small and doing just one thing. One PR can consist of as many commits as required. In this case, adding a new section isn't directly connected to the main purpose of this commit (adding support for .aws/credentials), so I suggest extracting it into a separate commit.